plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/AbstractResourceMojo.java

```diff
@@ -72,11 +73,11 @@ protected static Template getSingletonTemplate(KubernetesList resources) {
         return null;
     }
 
-    protected void writeResources(KubernetesList resources, ResourceClassifier classifier) throws MojoExecutionException {
+    protected void writeResources(KubernetesList resources, ResourceClassifier classifier, boolean generateRoute) throws MojoExecutionException {
         // write kubernetes.yml / openshift.yml
         File resourceFileBase = new File(this.targetDir, classifier.getValue());
 
-        File file = writeResourcesIndividualAndComposite(resources, resourceFileBase, this.resourceFileType, log);
+        File file = writeResourcesIndividualAndComposite(resources, resourceFileBase, this.resourceFileType, log, generateRoute);
```

Comment:
Nitpick: You're passing `boolean` from here which is being received as `Boolean` from `writeResourcesIndividualAndComposite`. I don't have any problem with this but It would be nice if we stay consistent with these.

Reply:
That's a typo on my end I will fix that